pkg/resources/resource_connection_aws.go

```diff
@@ -132,13 +132,13 @@ func connectionAwsRead(ctx context.Context, d *schema.ResourceData, meta interfa
 	// 	return diag.FromErr(err)
 	// }
 
-	if err := d.Set("assume_role_arn", s.AssumeRoleArn.String); err != nil {
-		return diag.FromErr(err)
-	}
+	// if err := d.Set("assume_role_arn", s.AssumeRoleArn.String); err != nil {
```

Comment:
Follow up task for this: https://github.com/MaterializeInc/terraform-provider-materialize/issues/629

Reply:
Oh I can tell you how to fix this: you need to set `--aws-external-id-prefix=<some dummy value>` in the Docker image.